sqlx-macros/src/database/postgres.rs

```diff
@@ -8,7 +8,7 @@ impl_database_ext! {
         i8,
         i16,
         i32,
-        u32,
+        sqlx::postgres::types::Oid,
```

Comment:
As a styling nit, this should be sorted with the other `Pg___` types below rather than stuck in the middle of a list of primitives.